### PR TITLE
Trigger Reminders cron

### PR DESCRIPTION
**Pull request by jeffreysilver**

### CODE REVIEW

File: __tests__/stopcovid/trigger_reminders/test_persistence.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import unittest
+import uuid
+
+
+from stopcovid import db
+from stopcovid.trigger_reminders.persistence import ReminderTriggerRepository
+from __tests__.utils.factories import make_drill_instance
+
+
+class TestReminderTriggerRepo(unittest.TestCase):
```
Comment (jeffreysilver) [inline __tests__/stopcovid/trigger_reminders/test_persistence.py:10]: Ported the ReminderTriggerRepo out of status, and into trigger_reminders.

File: __tests__/stopcovid/status/test_drill_instances.py [hunk 2]
```diff
@@ -17,6 +16,7 @@
 from stopcovid.drills.drills import Drill, Prompt
 from stopcovid.status.drill_instances import DrillInstanceRepository, DrillInstance
 from stopcovid import db
+from __tests__.utils.factories import make_drill_instance
```
Comment (pdsarin) [inline __tests__/stopcovid/status/test_drill_instances.py:19]: yay!

File: serverless.yml [hunk 3]
```diff
@@ -102,6 +102,12 @@ functions:
     events:
       - schedule: cron(5 17 * * ? *)
 
+  triggerReminders:
+    handler: aws_lambdas/trigger_reminders.handler
+    timeout: 60
+    events:
+      - schedule: rate(5 minutes)
```
Comment (pdsarin) [inline serverless.yml:109]: Should probably be daytime hours only. A cron expression would likely capture the running hours better.

File: stopcovid/trigger_reminders/trigger_reminders.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+import json
+import boto3
+import os
+
+from stopcovid.status.drill_instances import DrillInstance, DrillInstanceRepository
+
+from . import persistence
+
+REMINDER_TRIGGER_FLOOR_MINUTES = 60 * 4
+REMINDER_TRIGGER_CEIL_MINUTES = 60 * 12
```
Comment (pdsarin) [inline stopcovid/trigger_reminders/trigger_reminders.py:11]: per my other comment about daytime hours only, make sure that the ceiling is high enough that we wouldn't miss a reminder after a night has elapsed.

Assuming send times from 1-8 pm eastern time, we probably want to do 24 hours.

File: stopcovid/trigger_reminders/trigger_reminders.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+                        },
+                    }
+                ),
+                "PartitionKey": drill.phone_number,
+            }
+            for drill in drills
+        ]
+        kinesis.put_records(Records=records, StreamName=f"command-stream-{self.stage}")
+
+    def trigger_reminders(self):
```
Comment (pdsarin) [inline stopcovid/trigger_reminders/trigger_reminders.py:49]: this method is very readable

File: serverless.yml (hunk 3 above)
Comment (jeffreysilver) [reply serverless.yml:109]: Great call out

**Pull request merged**